x-pack/plugins/siem/public/store/timeline/reducer.test.ts

```diff
@@ -704,26 +740,86 @@ describe('Timeline', () => {
 
       const andProviderToAdd: DataProvider = {
         and: [],
-        id: '568',
+        id: '569',
         name: 'And Data Provider',
         enabled: true,
         queryMatch: {
-          field: '',
-          value: '',
+          field: 'happy',
+          value: 'andrewG',
           operator: IS_OPERATOR,
         },
+        excluded: false,
+        kqlQuery: '',
+      };
+      // temporary, we will have to decouple DataProvider & DataProvidersAnd
+      // that's bigger a refactor than just fixing a bug
```

Comment:
Thanks for these additional tests and clarifying comment. Definitely agree and something we can address later. 👍